features/utils.js

```diff
@@ -229,6 +224,24 @@ exports.sendCovid19CaseReport = async () => {
   console.log('Covid19 Case report successfully sent')
 }
 
+exports.sendCovid19CaseOutcome = async () => {
+  // Check whether the case outcome exists in DHIS
+  let response = await getDhisEntity(patientId2)
+
+  if (response.status == 200 && response.data.trackedEntityInstances.length) {
```

Comment:
@bradsawadye Is there a specific reason for using double equals in this file instead of triple?
Usually, triple is preferred as it's more specific.